PlayFabSdk/tests/.gitignore

```diff
@@ -0,0 +1 @@
+/testTitleData.js
```

Comment:
testTitleData.json?

Reply:
Also, this can probably be in the top level .gitignore...

Reply:
Yes, this allows you to store your testTitleData.json and this won't get propagated to any repositiory. 
By default, testTitleData.json gets read from ./tests folder. You can still specify PF_TEST_WHATEVER enveronmental variable like you did before. 
Ideally, we should add generator for the testTitleData if nothing is found.